### PR TITLE
Doc housekeeping

### PR DESCRIPTION
**Pull request by Vyrastas**
- Added an AUTHORS file, mimicked OpenThread and used the two contributing companies so far #48 
- Revised the LICENSE file to include Copyright and switched the content to the Apache boilerplate notice (the full license text is not necessary) #48 
- Fixed a broken documentation link #157

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,202 +1,14 @@
+Copyright (c) 2016, The OpenThread Authors.
+All rights reserved.
 
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```
Comment (darconeous) [inline LICENSE]: I think we really should be including our own copy of the license, not linking to a copy of it.

File: third_party/openthread/src/ncp/PROTOCOL.md [hunk 2]
```diff
@@ -1,7 +1,4 @@
 Spinel NCP Protocol Documentation
 =================================
 
-This document has moved into the top level `doc` folder and renamed
-[`draft-spinel-protocol.txt`](../../doc/draft-spinel-protocol.txt)
-and [`draft-spinel-protocol.html`](../../doc/draft-spinel-protocol.html).
-
+This document has moved to the [OpenThread repository](https://github.com/openthread/openthread/tree/master/doc).
```
Comment (darconeous) [inline third_party/openthread/src/ncp/PROTOCOL.md]: This file should likely just be removed.

File: LICENSE (hunk 1 above)
Comment (Vyrastas) [inline LICENSE]: Apologies, I got it confused with the boilerplate thing that goes in each individual file.  I've reverted, with the include of the copyright at the top and removed the appendix at the end that just had the instructions on applying the license.

File: third_party/openthread/src/ncp/PROTOCOL.md (hunk 2 above)
Comment (Vyrastas) [inline third_party/openthread/src/ncp/PROTOCOL.md]: Done

**Comment by darconeous** [comment]
The removal of `PROTOCOL.md` may require removing that file from `third_party/Makefile.am`. If there is a travis build failure, that's likely why.

**Comment by Vyrastas** [comment]
Yeah it was in the src/ncp-spinel one.  There was also a link on the Readme to it, changed that.

**Pull request merged**